march_joint_inertia_controller/src/inertia_estimator.cpp

```diff
@@ -0,0 +1,219 @@
+// Copyright 2020 Project March.
+#include <cmath>
+#include <hardware_interface/joint_command_interface.h>
+#include <hardware_interface/joint_state_interface.h>
+#include "march_joint_inertia_controller/inertia_estimator.h"
+#include "ros/ros.h"
+#include "std_msgs/Float64.h"
+
+std::vector<double> absolute(const std::vector<double>& a)
+{
+  std::vector<double> b;
+  b.resize(a.size());
+  for (size_t i = 0; i < a.size(); ++i)
+  {
+    b[i] = abs(a[i]);
+  }
+  return b;
+}
+
+double mean(const std::vector<double>& a)
+{
+  double sum = 0.0;
+  for (const double x : a)
+  {
+    sum += x;
+  }
+  if (a.size() == 0.0)
+  {
+    return 0.0;
+  }
+  return sum / a.size();
```

Comment:
I just thought if this now, but I you don't want to have the edge case, you can do as follows. It is up to you what you think is better code.
```suggestion
  for (const double x : a)
  {
    mean+= x / a.size();
  }
  return mean;
```

Reply:
I think I'll stick with the edge case for now, but thank you for the suggestion